datastore/db/bytebase/FertiScan/update_inspection_function.sql

```diff
@@ -525,9 +534,13 @@ BEGIN
         registration_number := p_input_json->'product'->>'registration_number';
 
         -- Insert organization and get the organization_id
-        INSERT INTO organization (information_id, main_location_id)
-        VALUES (company_info_id, NULL) -- TODO: main_location_id not yet handled
-        RETURNING id INTO organization_id;
+        IF company_info_id IS NOT NULL THEN
+            INSERT INTO organization (information_id, main_location_id)
+            VALUES (company_info_id, NULL) -- TODO: main_location_id not yet handled
+            RETURNING id INTO organization_id;
+        ELSE
+            organization_id := NULL;
```

Comment:
Should it be the manufacturer? If both are null, throw a Warning/Error? How do we plan to handle this edge case?

Reply:
It should be manufacturer. I changed it. 

The company (where the sample has been taken) doesn't affect the fertilizer object itself. So when it is null, it is simply ignored. Unless you had another idea in designing the schema.